ui/src/app/shared/components/pagination-panel.tsx

```diff
@@ -22,6 +22,14 @@ export class PaginationPanel extends React.Component<{pagination: Pagination; on
                     }>
                     Next page <i className='fa fa-chevron-right' />
                 </button>
+                {this.props.pagination.limit ? (
+                    <>
+                        <span className={'fa fa-exclamation-triangle'} style={{color: '#d7b700'}} />
+                        There is no global Workflow sort when pagination is enabled
```

Comment:
I don't think this is particularity good copy. Any suggestions?

Reply:
`Workflows cannot be sorted when paginated`